src/commands/pg/privatelink/index.ts

```diff
@@ -28,10 +28,10 @@ export default class EndpointsIndex extends BaseCommand {
 
     if (res.status === 'Provisioning') {
       this.log()
-      this.log(`The Trusted VPC Endpoint is now being provisioned for ${color.cyan(database)}.`)
-      this.log(`Run ${color.cyan('heroku endpoints:wait')} to check the creation process.`)
+      this.log(`The privatelink endpoint is now being provisioned for ${color.cyan(database)}.`)
+      this.log(`Run ${color.cyan('heroku pg:privatelink:wait -a APP')} to check the creation process.`)
```

Comment:
At this point, do we _know_ the app? Could we interpolate it here for them for easy copy/paste?